### PR TITLE
config settings have no impact

### PR DESCRIPTION
**Pull request by dcsan**
I ran a few examples with different settings but the only thing that seems to make any difference is `maxTokenSize`

the configs I ran are as below and they all come out with the same number of chunks.
you can see the outputs in data/docs 

am i doing something wrong?

```ts

    v1: {
        logging: true,
        maxTokenSize: 800,
        similarityThreshold: .25,             // higher value requires higher similarity to be included (less inclusive)
        dynamicThresholdLowerBound: .2,        // lower bound for dynamic threshold
        dynamicThresholdUpperBound: .9,        // upper bound for dynamic threshold
        numSimilaritySentencesLookahead: 3,
        combineChunks: true,
        combineChunksSimilarityThreshold: 0.3, // lower value will combine more chunks (more inclusive)
        onnxEmbeddingModel: "Xenova/all-MiniLM-L6-v2",
        onnxEmbeddingModelQuantized: true,
    },
    v2: {
        logging: true,
        maxTokenSize: 800,
        similarityThreshold: .5,             // higher value requires higher similarity to be included (less inclusive)
        dynamicThresholdLowerBound: .2,        // lower bound for dynamic threshold
        dynamicThresholdUpperBound: .9,        // upper bound for dynamic threshold
        numSimilaritySentencesLookahead: 3,
        combineChunks: true,
        combineChunksSimilarityThreshold: 0.5, // lower value will combine more chunks (more inclusive)
        onnxEmbeddingModel: "Xenova/all-MiniLM-L6-v2",
        onnxEmbeddingModelQuantized: true,
    },
    v3: {
        logging: true,
        maxTokenSize: 800,
        similarityThreshold: .75,             // higher value requires higher similarity to be included (less inclusive)
        dynamicThresholdLowerBound: .2,        // lower bound for dynamic threshold
        dynamicThresholdUpperBound: .9,        // upper bound for dynamic threshold
        numSimilaritySentencesLookahead: 3,
        combineChunks: true,
        combineChunksSimilarityThreshold: 0.75, // lower value will combine more chunks (more inclusive)
        onnxEmbeddingModel: "Xenova/all-MiniLM-L6-v2",
        onnxEmbeddingModelQuantized: true,
    },
```

### CODE REVIEW

File: data/docs/aapl-2024-q2.size-300.sim-0.5.comb-0.3.dyn-0.2-0.9.chunks.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "We're guiding to again to a very high level of gross margins, 45.5% to 46.5%. Within that guidance, we expect memory to be a slight headwind, not a very large one, but a slight headwind. And the same applies for foreign exchange. Foreign exchange will have a negative impact sequentially of about 30 basis points. Atif Malik\n\nThank you. Suhasini Chandramouli\n\nThank you, Atif. A replay of today's call will be available for two weeks on Apple podcasts as a webcast on apple.com/investor and via telephone. The number for the telephone replay is 866-583-1035. Please enter confirmation code 0467138 followed by the pound sign. These replays will be available by approximately 5:00 P.M. Pacific Time today. Members of the press with additional questions can contact Josh Rosenstock at 408-862-1142, and financial analysts can contact me, Suhasini Chandramouli, with additional questions at 408-974-3123. Thank you again for joining us. "
+  ],
+  "count": 38,
```
Comment (dcsan) [inline data/docs/aapl-2024-q2.size-300.sim-0.5.comb-0.3.dyn-0.2-0.9.chunks.json:42]: config settings at the bottom

**Comment by justin-parker** [comment]
@dcsan try setting `combineChunks: false` if you want distinct separate chunks. When `combineChunks` is set to `true`, it will attempt to merge neighboring chunks together until the max token size is met as a last operation before returning the final chucked array (it will still attempt to keep the initial chunks as semantically close as possible based on all other config settings).  I will run some tests and review your submitted files when I get back from summer vacation.

**Comment by jparkerweb** [comment]
@dcsan 
change your runner.js file to import/call `chunkit`, not `cramit`. It appears you created your `runner.js` by using some of the code from the `example2.js` file, which calls `cramit`.

example chunkit usage:

```javascript
import { chunkit } from 'semantic-chunking';

let frogText = "A frog hops into a deli and croaks to the cashier, \"I'll have a sandwich, please.\" The cashier, surprised, quickly makes the sandwich and hands it over. The frog takes a big bite, looks around, and then asks, \"Do you have any flies to go with this?\" The cashier, taken aback, replies, \"Sorry, we're all out of flies today.\" The frog shrugs and continues munching on its sandwich, clearly unfazed by the lack of fly toppings. Just another day in the life of a sandwich-loving amphibian! 🐸🥪";

async function main() {
    let myFrogChunks = await chunkit(frogText, { maxTokenSize: 65 });
    console.log("myFrogChunks", myFrogChunks);
}
main();
```

`cramit` is a quick a dirty method when you just need high density chunks. Read the docs here:
[🧼 Cramit Docs](https://github.com/jparkerweb/semantic-chunking?tab=readme-ov-file#cramit----the-quick--dirty).

**Comment by jparkerweb** [comment]
@dcsan closing as you were calling the wrong method. Open a new issue if you run into anything else or have more questions.

**Pull request closed without merging**